x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/scalar/string/regex/WildcardLike.java

```diff
@@ -51,11 +50,23 @@ also act on a constant (literal) expression. The right-hand side of the operator
 
         <<load-esql-example, file=string tag=likeEscapingSingleQuotes>>
 
-        <<load-esql-example, file=where-like tag=likeListDocExample>>
-
         To reduce the overhead of escaping, we suggest using triple quotes strings `\"\"\"`
 
         <<load-esql-example, file=string tag=likeEscapingTripleQuotes>>
+
+        ```{applies_to}
+        stack: ga 9.1
+        serverless: ga
+        ```
+        Both a single pattern or a list of patterns are supported. If a list of patterns is provided,
+        the expression will return true if any of the patterns match.
+
+        ```{applies_to}
+        stack: ga 9.1
+        serverless: ga
+        ```
```

Comment:
```suggestion
```
![screenshot_2025-06-29_at_8 14 41___pm_720](https://github.com/user-attachments/assets/f3141226-7885-490e-959c-6d49626dd049)

Looking at your preview @julian-elastic I don't think you need this twice? WDYT?